src/components/CodeBlock/index.js

```diff
@@ -27,7 +29,7 @@ export const LineNo = styled.span`
 
 const CopyCode = styled.button`
   position: absolute;
-  right: 0;
+  right: 5em;
```

Comment:
This will affect all the places where this component is used, have you confirmed if it is looking fine in all the other places and in all views?